src/Pages/ara/Main.js/Feed.js

```diff
@@ -0,0 +1,155 @@
+import React from "react";
+import "./Feed.scss";
+import Comment from "./Comment.js";
+
+const FEEDINFO = [
```

Comment:
```suggestion
const FEED_INFO = [
```
- 상수 변수는 잘 선언해주셨으나, 대문자 스네이크 케이스로 작성해주세요.
- 이 내용이 컴포넌트보다 중요한 정보는 아니므로 컴포넌트 아래로 자리를 옮겨주세요.